docs/dev/security/index.md

```diff
@@ -0,0 +1,20 @@
+# Security at Aragon
+
+The developers supporting the Aragon project take security and their users' wellbeing very seriously.
+
+If you have found a potential vulnerability, please review the following instructions for responsibly disclosing it to the team.
+
+## Reporting
+
+- In order to report a vulnerability, please write an email to `security@aragon.org` with [BUG BOUNTY] in the subject of the email.
```

Comment:
Would a more generic subject like "SECURITY DISCLOSURE" be more appropriate, since some vuln reports might not be covered by the bug bounty?

Reply:
Ah yes, good catch. Missed that!